Model/ModelManager.php

```diff
@@ -376,7 +386,7 @@ public function getNormalizedIdentifier($entity)
         }
 
         // the entities is not managed
-        if (!$entity /*|| !$this->getEntityManager($entity)->getUnitOfWork()->isInIdentityMap($entity) // commented for perfomance concern */) {
+        if (!$entity || !$this->getEntityManager($entity)->getUnitOfWork()->isInIdentityMap($entity)) {
```

Comment:
(@doctrine is a parameter in the setEntityManager method. We don't store the doctrine service in the admin, just the EM. Maybe it could be called `setEntityManagerBasedOnDoctrineRegistry()`, but it's not something high on the 'to do' list of changes! Allowing users to edit objects without getting an error message or being bounced to edit// is the matter in hand :-) )